CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,40 @@
+# Fuel Rats Contributor Code of Conduct
+
+Please read through this document and understand the specifics.  If you do not, feel free to ask another contributor, or contact a member of the Moderation team.
+
+This policy is in place to establish behavioral expectations and guidelines to ensure an open, tolerant, and professional environment for all participants.  We believe anyone participating in our projects deserve a harassment-free experience, regardless of age, caste, gender, gender identity or expression, culture ethnicity, disability, language, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, or technical ability.  Discrimination based on any of these protected characteristics will not be tolerated.
+
+This policy is NOT intended to supersede the Fuel Rats [Terms of Service](https://fuelrats.com/terms-of-service), but as a supplement to cover non-rescue contributions.
+
+## Standards
+
+We hope all members strive to go beyond examples laid out here and actively create a positive and respectful atmosphere.
+
+* **Foster a friendly, judgement-free environment for all.**  This project houses international contributors from all walks of life, with varying experience.
+* **Keep an open-mind, and respect differing viewpoints.** Lest we forget the professional attitude best displayed in our work.
+* **Be professional and considerate.** Discuss and resolve conflicts in an open, civil manner.  Disagreements are inevitable.
+* **Show patience and be helpful.**  We all make mistakes.  Playing the 'blame game' helps no one.
+
+#### Negative Behavior Examples
+
+* Forcing others to participate
+* Public or private harassment
+* Trolling/Personal attacks
+* 'Doxxing' or making a person's private information available without explicit permission
+* Conduct considered inappropriate for a professional environment.
+* Conduct considered unbecoming by any member of the 'Tech Rat Moderation' team.
+
+## Repository Admins/Owners
+
+Repository Owners are expected to use this document or provide a link to it in their repository. Additional items may be added, but no thing may be added to reduce the effectiveness of this document or its intent.
+
+Admins/Owners have the responsibility to remove and report comments, code, commits, issues, or any other post that does not align with this policy, and report it to a member of the 'Tech Rat Moderation' team.
```

Comment:
I'd prefer the idea of using "project lead" if you mean owners, or "maintainer" for admins/org members.

Reply:
I agree with using the language "Project lead" as well, the use of admin can be confusing and may be thought to refer to net admins, or ops.